### PR TITLE
feat: add support for TLSv1.3

### PR DESCRIPTION
**Pull request by yugasun**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+# refer to https://github.com/gliderlabs/docker-alpine/issues/466
+FROM alpine:3.9
 
 LABEL maintainer="Wang Shaobo <fireshooter@163.com>, xuexb <fe.xiaowu@gmail.com>"
 
 ENV LANG en_US.UTF-8
 RUN sed -i 's/dl-cdn.alpinelinux.org/mirrors.aliyun.com/g' /etc/apk/repositories
 
-ENV NGINX_VERSION 1.11.2
+ENV NGINX_VERSION 1.14.2
```
Comment (xuexb) [inline Dockerfile]: Nginx 版本不能升级，因为使用了 [echo-nginx-module](https://github.com/openresty/echo-nginx-module#compatibility) ，该模块目前最高支持到 1.11.2 。

Comment (yugasun) [reply Dockerfile]: I have tested `echo-nginx-module`, it is still working with nginx@1.14.2, you can test it.
The critical error is if I decreate nginx version to 1.11.2, installing `lua-nginx-module` will fail with errors.

Comment (jpcn19) [reply Dockerfile]: 666，xiaowu said

Comment (xuexb) [reply Dockerfile]: @yugasun 
```bash
make
cd test && docker-compose up
```

--->>>

```
test-nginx | 2019/02/25 23:45:48 [alert] 1#1: detected a LuaJIT version which is not OpenResty's; many optimizations will be disabled and performance will be compromised (see https://github.com/openresty/luajit2 for OpenResty's LuaJIT or, even better, consider using the OpenResty releases from https://openresty.org/en/download.html)
```

File: scripts/install.sh [hunk 2]
```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+# Install LUAJIT
+apk add --no-cache luajit libwebp-tools
```
Comment (xuexb) [inline scripts/install.sh]: https://github.com/shangxianapp/docker-nginx-alpine/blob/98e82e63587b4ec26b70d194c3b71a1c702dc4c2/Dockerfile#L14-L15

删除 Dockerfile 里的吧，全部在 sh 里处理。

File: Dockerfile (hunk 1 above)
Comment (yugasun) [reply Dockerfile]: This is not problem about our installation steps, but about `openresty`, `openresty` official suggest we install [Openresty's luajit2](https://github.com/openresty/luajit2), so when we use openresty without official version, this alert will come out. To solve this, we can just change to openresty's version.

**Pull request merged**